cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,25 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signUp.pageObject';
+
+const signUpPage = new SignUpPageObject();
+
 describe('User', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
-
+  it('should be able to follow the another user', () => {
```

Comment:
The test description 'should be able to follow the another user' contains a grammatical error. Consider changing it to 'should be able to follow another user' for clarity.